lib/src/configuration.dart

```diff
@@ -295,3 +317,16 @@ class RealmSchema extends Iterable<SchemaObject> {
   @override
   SchemaObject elementAt(int index) => _schema.elementAt(index);
 }
+
+/// The signature of a callback that will be invoked if a client reset error occurs for this [Realm].
+///
+/// Currently, Flexible sync only supports the Manual Recovery.
```

Comment:
The docs should probably mention `[ManualSyncClientResetHandler]` so that it's explicit what is the type users are supposed to use.

Reply:
fixed